src/components/interface/LogoutModal.vue

```diff
@@ -12,18 +12,18 @@
         </div>
       </div>
 
-      <div class="flex items-center pb-4 mx-4">
+      <div class="mr-4 text-center">
```

Comment:
I do like the idea of having an apparent log out confirmation button, but I believe that consistency throughout the UI is best for the user (horizontal button layout similar to, if not exactly like, what you have for the `ConfirmationModal`)

<img width="520" alt="Screen Shot 2021-09-20 at 14 22 59" src="https://user-images.githubusercontent.com/22848332/134077669-236e374f-eaa9-48d6-9d6e-5178a14cf708.png">